account_block_account/i18n/nl.po

```diff
@@ -0,0 +1,34 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* account_block_account
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-02-20 15:12+0000\n"
+"PO-Revision-Date: 2018-02-20 15:12+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: account_block_account
+#: code:addons/account_block_account/models/blocked_accounts_mixin.py:37
+#, python-format
+msgid "%s is blocked!"
+msgstr "%s is geblokeerd!"
+
+#. module: account_block_account
+#: model:ir.model,name:account_block_account.model_blocked_accounts_mixin
+msgid "A model that understands about blocked accounts"
+msgstr "Een model dat met beklokkeerde rekeningen om kan gaan"
```

Comment:
geblokkeerde (wow, this one :)

Reply:
we could do funny stuff about how this might mean time related stuff

Reply:
Yes, next time let me also clockate more